website/docs/cli/commands/list/list-stacks.mdx

```diff
@@ -44,3 +44,54 @@ atmos list stacks -c vpc
 | Flag             | Description                                                                                                                                                                                                                               | Alias | Required |
 |------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------|----------|
 | `--component`    | Atmos component                                                                                                                                                                                                                         | `-c`  | no       |
+
+## Display Format
+
+The command output can be customized using template variables in your `atmos.yaml` configuration. Here's an example of how to configure the display format to add a custom field called “Tenant”:
+
+```yaml
+stacks:
+  display:
+    columns:
+      - name: Stack
+        value: '{{ .atmos_stack }}'
+      - name: Tenant
+        value: '{{ index .vars "tenant" }}'
+```
+
```

Comment:
Note, that in this example, it assumes that every component defines `vars.tenant` and `vars.environment`.